app/src/main-process/menu/build-default-menu.ts

```diff
@@ -272,6 +272,11 @@ export function buildDefaultMenu(sharedProcess: SharedProcess): Electron.Menu {
         accelerator: 'CmdOrCtrl+Shift+C',
         click: emit('compare-branch'),
       },
+      {
+        label: __DARWIN__ ? 'Create Pull Request' : 'Create Pull Request',
```

Comment:
I'm not sure. I kinda thought Pull Request was like a common noun that we came up with. I don't know tbh.